CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ set(root_components_needed RIO Tree)
 set(root_min_version 6.28.04)
 if(ENABLE_RNTUPLE)
   list(APPEND root_components_needed ROOTNTuple)
-  set(root_min_version 6.32)
+  set(root_min_version 6.34)
```

Comment:
This is technically not necessary, as things still build with 6.32. However, the files that are produced have no guarantees wrt backwards or forwards compatibility (from ROOT). So the question is whether we want to be on the safe side here or not. Bumping this has quite some implications for our CI, as many workflows still run on older versions of ROOT.